frontend-web/src/coffee-call/coffee-call.tsx

```diff
@@ -30,7 +36,7 @@ export const CoffeeCall: Component = () => {
         </Show>
       </div>
       <ul>
-        <For each={protocol.messages()}>{(message) =>
+        <For each={allMessages()}>{(message) =>
```

Comment:
Belongs to suggestion above.

```suggestion
        <For each={messages()}>{(message) =>
```